zktop.py

```diff
@@ -93,14 +95,14 @@ def __init__(self, session, server_id):
 class ZKServer(object):
     def __init__(self, server, server_id):
         self.server_id = server_id
-        self.host, self.port = server.split(':')
+        self.host, self.port = server.split(':') if ':' in server else (server, ZK_DEFAULT_PORT) # fallback to default if user doesn't specify port number
```

Comment:
Can you reformat this (throughout the changes) - line is long and extending beyond the screen on the left. I typically prefer the 

if foo
  bar

form.